plugins/search/providers/elastic_search/lib/kElasticSearchManager.php

```diff
@@ -233,8 +234,16 @@ private function shouldSyncElastic($object)
 
     private function shouldSkipSaveToSphinxLog($object, &$command)
     {
-        // limit the number of large SQLs to 1/min per object, since they load the sphinx log database and elastic servers
-	$commandSize = strlen($command);
+		$commandSize = strlen($command);
+		
+		// limit object index size up to 1MB due to mysql current 'max_allowed_packet' config which is 1MB
+		if ($commandSize > self::SQL_MAX_ALLOWED_PACKET)
+		{
+			KalturaLog::log('skipping saving elastic sphinxLog sql for [' . $object->getId() . '] - command size [' . $commandSize .'] larger than 1MB');
```

Comment:
wouldn't that make Elasticsearch clusters inconsistent when using Multi DC?
Also, as this is a tunable value (OnPerms and Regional Clouds set that to 16MB) on the sql side, should this be adjustable as well?